middleware/localNamingConventions.js

```diff
@@ -12,8 +27,17 @@ function applyLocalNamingConventions(req, res, next) {
 
   // loop through data items and flip relevant number/street
   res.data.filter(function(place){
-    // only relevant for German addresses
-    if( place.parent.country_a.indexOf('DEU') === -1 ){ return false; }
+    // relevant for some countries
+    var flip = false;
+    for (var i=0; i<place.parent.country_a.length; i++) {
+      var country_a = place.parent.country_a[i];
+      if( flipNumberAndStreetCountries.indexOf(country_a) !== -1 ) {
+	flip = true;
+	break;
+      }
+    }
+    if (!flip){ return false; }
```

Comment:
Again with lodash, I think this can be written as

```
var flip = place.parent.country_a.some(function(country) {
  return _.includes(flipNumberAndStreetCountries, country);
});